parmed/amber/_amberparm.py

```diff
@@ -379,7 +380,8 @@ def from_structure(cls, struct, copy=False):
                 dt.per = 1.0
         inst.remake_parm()
         inst._set_nonbonded_tables(nbfixes)
-        if inst.pointers['NCOPY'] >= 2:
+        n_copy = inst.pointers.get('NCOPY', 1)
+        if n_copy >= 2:
```

Comment:
if inst.pointers.get('NCOPY', 1):
Won't work since it's alway True. :p
Do you mean?
if inst.pointers.get('NCOPY', 1) >= 2:
Hai
> On Aug 17, 2016, at 11:04 AM, Jason Swails <notifications@github.com> wrote:
> 
> In parmed/amber/_amberparm.py:
> 
> > @@ -379,7 +380,8 @@ def from_structure(cls, struct, copy=False):
> >                  dt.per = 1.0
> >          inst.remake_parm()
> >          inst._set_nonbonded_tables(nbfixes)
> > -        if inst.pointers['NCOPY'] >= 2:
> > +        n_copy = inst.pointers.get('NCOPY', 1)
> > +        if n_copy >= 2:
> It could have been done
> 
> if inst.pointers.get('NCOPY', 1):
>     inst._label_alternatives()
> but this way gives me higher coverage :).
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>